lib/whitehall/authority/rules/person_rules.rb

```diff
@@ -0,0 +1,11 @@
+module Whitehall::Authority::Rules
+  PersonRules = Struct.new(:actor, :subject) do
+    def can?(_action)
+      if subject.slug == "boris-johnson"
+        actor.vip_editor?
```

Comment:
Should we also add GDS Admins here, as GDS's Content Designers often change people/roles during reshuffles?

Reply:
Yep, sounds good 👍